source/AI.cpp

```diff
@@ -4928,6 +4929,35 @@ void AI::CacheShipLists()
 
 
 
+void AI::RegisterDerivedConditions(ConditionsStore& conditions)
+{
+	// Special conditions about system hostility.
+	conditions["government strength: "].ProvidePrefixed([this](const ConditionEntry &ce) -> int64_t {
+		string name = ce.NameWithoutPrefix();
+		int64_t strength = 0;
+		for(const Ship *ship : governmentRosters[GameData::Governments().Get(name)])
```

Comment:
Make it like the others
```suggestion
		const Government *gov = GameData::Governments().Get(ce.NameWithoutPrefix());
		int64_t strength = 0;
		for(const Ship *ship : governmentRosters[gov])
```